File: fizzbuzz.js
#!/usr/bin/env node
"use strict";
var sys = require('sys');
(function () {
  var i, msg;
  for (i = 1; i <= 100; i += 1) {
    msg = '';

    if (0 === (i % 3)) { msg += "Fizz"; }
    if (0 === (i % 5)) { msg += "Buzz"; }
    if (msg.length > 0) { console.log(msg); }
  }
}());

File: fizzbuzz.rb
#!/usr/bin/env ruby
(1..100).each do |x|
  print "Fizz" if 0 == (x % 3)
  print "Buzz" if 0 == (x % 5) 
  print "\n" if 0 == (x % 3) || 0 == (x % 5)
end
